tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fvm_overview.md

```diff
@@ -0,0 +1,245 @@
+# New Feature: Finite Volume Method (FVM)
+
+!---
+
+## Advantages of Finite Volume Method
+
++(1) Flexible:+ Works with arbitrary cell types (even if they are mixed in the same problem)
+
++(2) Inherently conservative:+ Uses the integral form of the PDEs. This makes it perfect for
+      applications where conservation laws need to be respected (fluid flows).
+
++(3) "Accurate":+ Can be second-order accurate in space.
```

Comment:
```suggestion
+(3) "Accurate":+ Can be second-order accurate in space.
```

Why the quotes?

Reply:
Because it is hard to go beyond that, while with FEM you can just increase the order of the approximating polynomial. So compared to FEM, it is not super accurate, but in general it is accurate enough.

Reply:
I'll remove the quotes :P